.travis.yml

```diff
@@ -10,14 +9,12 @@ php:
 env:
   global:
     - TEST_COMMAND="composer test"
-    - SYMFONY_VERSION=2.7.*
+    - SYMFONY_VERSION=2.8.*
```

Comment:
i agree that we should change this, but maybe to 3.0 right away. we need to adjust the symfony version for the matrix lines below as well so we still test 2.7, 2.8 and 3.0